src/lib/memfs.ts

```diff
@@ -45,7 +45,7 @@ const isFilePath = /^[./]/;
 // Patch to support require() calls within test chunks (eg. dynamic-imports)
 const { _load } = Module as typeof NodeModule;
 (Module as typeof NodeModule)._load = function _memoryLoad(request, parent) {
-	if (hasMemFsInCallStack(parent)) {
+	if (typeof parent !== 'undefined' && hasMemFsInCallStack(parent)) {
```

Comment:
Can you see if these changes fix it?

https://github.com/privatenumber/instant-mocha/pull/15

Reply:
It did, will remove the commit